docs/eventing/getting-started.md

```diff
@@ -14,22 +14,21 @@ transport the events.
 ## Creating a Knative Eventing namespace
 
 Namespaces are used to group together and organize your Knative resources.
```

Comment:
It looks like this is unrelated cleanup?

If you want to do this file on its own, I could probably LGTM this on a single go-round.

Reply:
Yeah it's unrelated clean up. I just see things sometimes as I'm working on something else and add them in.
I'll maybe try to make smaller clean up PRs instead but can't promise I won't fix some things as I notice them 😂